tests/tests/fv3_regional_natl_c192

```diff
@@ -0,0 +1,62 @@
+###############################################################################
+#
+#  FV3 CCPP regional c192 test for North Atlantic domain
+#
+###############################################################################
+
+export TEST_DESCR="Compare FV3 CCPP regional c192 results for North Atlantic domain with previous trunk version"
+
+export CNTL_DIR=fv3_regional_natl_c192
+
+export LIST_FILES="atmf024.nc \
+                   sfcf024.nc \
+                   RESTART/coupler.res \
+                   RESTART/fv_core.res.nc \
+                   RESTART/fv_core.res.tile1.nc \
+                   RESTART/fv_srf_wnd.res.tile1.nc \
+                   RESTART/fv_tracer.res.tile1.nc \
+                   RESTART/phy_data.nc \
+                   RESTART/sfc_data.nc"
+
+export_fv3
+
+export TASKS=1280
```

Comment:
We may not be able to use that much of nodes (threads=2) in RT, can you reduce this to a low resolution test? There is a requirement that every test needs to finish within 30mins including data comparison withe baseline and whole RT finishes  in 2-3 hours including waiting time in the queue

Reply:
Have substantially reduced the nodes/PEs down to <= 360 PEs used for these new added regression tests. Also, forecast hour are reduced together with time step adjustment enable the forecast to finish around 6 mins.